pypesto/optimize/optimizer.py

```diff
@@ -613,7 +619,7 @@ def minimize(
                 "This optimizer requires an installation of cma.")
 
         result = cma.CMAEvolutionStrategy(
-            x0, sigma0, inopts=self.options,
```

Comment:
this you can actually leave in. it is called "trailing comma" and is a good feature of python, as it allows to easier append arguments a line below.

Reply:
Done